### ISSUE TITLE
pagination links do not meet minimum color contrast requirements

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6760]
### Descriptive summary

On dev.nurax.samvera.org (Hyrax 5.0.0), the pagination links do not meet WCAG AA minimum color contrast requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to https://dev.nurax.samvera.org/catalog?locale=en&q=&search_field=all_fields&utf8=%E2%9C%93&view=list and run the Site Improve browser extension

![pagination_color_contrast](https://github.com/samvera/hyrax/assets/32644254/09910921-10d7-4de0-86c6-545797c1acaf)

Hyrax (and nurax) should be accessible to users out of the box.

**Comment by rjkati** [comment]
This issue occurs on the following pages:

- [Search Results List, Gallery, Masonry and Slideshow Views](https://dev.nurax.samvera.org/catalog?locale=en&q=&search_field=all_fields&utf8=%E2%9C%93&view=list)
- [List of Users](https://dev.nurax.samvera.org/users?locale=en)
- [User Dashboard for non-admin user](https://dev.nurax.samvera.org/dashboard?locale=en)
- [My Collections](https://dev.nurax.samvera.org/dashboard/my/collections?locale=en)
- [All Collections or Managed Collections](https://dev.nurax.samvera.org/dashboard/collections?locale=en)
- [My Works](https://dev.nurax.samvera.org/dashboard/my/works?locale=en)
- [All Works or Managed Works](https://dev.nurax.samvera.org/dashboard/works?locale=en)
- [Workflow Roles](https://dev.nurax.samvera.org/admin/workflow_roles?locale=en)

Hopefully, a CSS tweak can fix all of these, but please comment here if it is preferable to create tickets for each occurrence and I'll do that.

**Comment by rjkati** [comment]
Closed as fixed via pagination changes as described in #6847
